docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -72,7 +90,7 @@ Note:
 * The schema file doesn't have to be called ``schema.cql``, this is just the default name. Any file name is supported (with any extension).
 
 Dropped columns
-^^^^^^^^^^^^^^^
+"""""""""""""""
```

Comment:
@denesb 
Sphinx reports a warning because of this change because such a heading is not supported; see https://sphinx-theme.scylladb.com/stable/examples/headings.html.
Did you want to nest "Dropped columns" under the "schema.cql" heading?

The heading syntax in this file is all mixed up because the page title follows the syntax of H2. The documentation builds without warnings as long as we don't make any changes to the hierarchy.

If the "Dropped columns" heading should be a lower lever heading than "schema.cql", unfortunately, we must:
- Revert the change on this line back to ^^^^^^^^^^^^^^^
- Open an issue to fix the heading syntax on this page, pointing out that "Dropped columns" is lower H than "schema.cql".
- Fix the headings on the entire page AFTER this PR is merged  (to avoid messing up the docs in this PR).

Reply:
I ended up using `~~~~~` and Sphinx seems to be happy with that.